apps/passport-client/src/worker/service-worker.ts

```diff
@@ -53,6 +59,18 @@ const EPHEMERAL_CACHE_RESOURCES = new Set([
   ...GENERATED_CHUNKS
 ]);
 
+/**
+ * Z cache is for Z API-relevant resources which do not change frequently, such
+ * as GPC artifacts, where we always use the cached version if available.  This
+ * cache is only updated if the artifact to be fetched is not in the
+ * cache. Previously cached versions are cleared for space-saving purposes.
```

Comment:
But the cache is never cleared as long as the version doesn't change, so it can be as large as the max number of artifacts ever used?  I.e. no LRU replacement or max size.  I think that's fine/correct, just that it should be clearly documented here.